### PR TITLE
test: add comprehensive tests for collections and view models

### PR DESCRIPTION
**Pull request by ValeraFinebits**
## Issue

N/A

## Description

- Add extensive unit tests for ManagedCollection and IncrementalLoadingCollection, covering filtering, sorting, reconciliation, and edge cases
- Introduce test doubles and helpers to improve test isolation and reliability
- Update test infrastructure: centralize test dependencies, add UI test project, and improve .gitignore
- Enhance coverage for ContactsPanelPageViewModel, including command and event handling
- Improve code quality and maintainability by enforcing analyzers and code style in test projects

## Pull Request Checklist

Please check if your pull request fulfills the following requirements:

- [x] The pull request is named according to the format: _[\<type\>](#types-of-pull-requests)(optional scope): \<description\>_
- [ ] The pull request is associated with a GitHub issue. Note: if possible use the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
- [x] [The pull request is labeled](#pull-request-labels)
- [x] [Affected platforms are labeled](#platform-labels)
- [x] Commits follow the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] There are no difficult to understand places left without comments
- [x] The changes do not generate new warnings

> [!NOTE]
>
> <details><summary><strong id="types-of-pull-requests">Types of pull requests</strong></summary>
>
> - `feat` - adding **new features**
> - `fix` - **bug** fixes
> - `test` - adding or correcting **tests**
> - `perf` - changes that improve **performance**
> - `refactor` - simple **rewriting** or **restructuring** of code without adding new features or fixing bugs
> - `style` - changes in **code styles** and no changes in logic
> - `build` - changes related to **the build of the project** and **dependencies**
> - `ci` - changes related to **continuous integration**
> - `docs` - changes in **documentation** or just **comments** in source code
> - `chore` - something that **doesn't fit** the other possible types
>
> </details>
>
> <details><summary><strong id="pull-request-labels">Pull request labels</strong></summary>
>
> - **`type/breaking-change`** - pull requests with changes that are **not backward compatible**
> - `type/build` - pull requests that change the **project's build** or **dependencies**
> - `type/chore` - pull requests **without** making **changes** to the code, project build, formatting, documentation, etc
> - `type/ci` - pull requests whose changes are related to **continuous integration**
> - `type/documentation` - pull requests that only change **documentation**
> - **`type/feature`** - pull requests that add **new features**
> - **`type/fix`** - pull requests that fix a **bug**
> - **`type/localization`** - pull requests that change **translation**
> - **`type/performance`** - pull requests that improve **performance**
> - `type/refactor` - pull requests that **refactor** a section of code
> - `type/style` - pull requests that change **code styles**
> - `type/test` - pull requests that add or correct **tests**
> - **`ignore-for-release`** - for pull requests that do **not need** to be appeared in **release notes**
>
> ---
> A pull request appears in the release notes if it has one of the labels: `type/breaking-change`, `type/feature`, `type/fix`, `type/localization`, `type/performance`
>
> </details>
>
> <details><summary><strong id="platform-labels">Affected platforms labels</strong></summary>
>
> - `platform/all` - pull requests that are related to the all platforms
> - `platform/android` - pull requests that are related to the Android platform
> - `platform/desktop` - pull requests that are related to the desktop
> - `platform/ios` - pull requests that are related to the iOS platform
> - `platform/macos` - pull requests that are related to the macOS platform
> - `platform/other` - pull requests that are related to an unknown platform
> - `platform/uwp` - pull requests that are related to the Universal Windows Platform
> - `platform/wasm` - pull requests that are related to the WebAssembly platform
> - `platform/winui` - pull requests that are related to the WinUI platform
>
> </details>
>

### CODE REVIEW

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 1]
```diff
@@ -0,0 +1,1287 @@
...
+            cts.Cancel();
+            var source = new PagedIntSource(Enumerable.Range(1, 100));
+            var collection = new IncrementalLoadingCollection<PagedIntSource, int>(source, cts);
+
+            var result = await collection.LoadMoreItemsAsync(10).AsTask().ConfigureAwait(false);
+
+            Assert.That(result.Count, Is.Zero);
+            Assert.That(collection.Count, Is.Zero);
+            Assert.That(collection.HasMoreItems, Is.False);
+            Assert.That(source.LoadCalls, Is.EqualTo(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Is.Zero`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 2]
```diff
@@ -0,0 +1,1287 @@
...
+        }
+
+        [Test]
+        public async Task RefreshAsyncOnEmptyCollectionForcesInitialLoad()
+        {
+            using var cts = new CancellationTokenSource();
+            var source = new PagedIntSource(Enumerable.Range(1, 50));
+            var collection = new TestIncrementalLoadingCollection(source, cts);
+
+            Assert.That(collection.Count, Is.EqualTo(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Empty);` for any `IEnumerable` collection, otherwise use `Is.Zero`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 3]
```diff
@@ -0,0 +1,1287 @@
...
+            using var cts = new CancellationTokenSource();
+            var source = new PagedIntSource(Enumerable.Range(1, 10));
+            var collection = new NegativePageSizeCollection(source, cts);
+
+            Exception? observed = null;
+            collection.OnError = ex => observed = ex;
+
+            var result = await collection.LoadMoreItemsAsync(1).AsTask().ConfigureAwait(false);
+
+            Assert.That(result.Count, Is.EqualTo(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Empty);` for any `IEnumerable` collection, otherwise use `Is.Zero`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 4]
```diff
@@ -0,0 +1,1287 @@
...
+        public void ConstructorWithoutSourceCreatesInstanceViaActivator()
+        {
+            using var cts = new CancellationTokenSource();
+
+            // This constructor uses Activator.CreateInstance<TSource>()
+            var collection = new IncrementalLoadingCollection<PagedIntSource, int>(cts);
+
+            Assert.That(collection, Is.Not.Null);
+            Assert.That(collection.HasMoreItems, Is.True);
+            Assert.That(collection.Count, Is.EqualTo(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Empty);` for any `IEnumerable` collection, otherwise use `Is.Zero`

File: src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ContactsPanelPageViewModelTests.cs [hunk 5]
```diff
@@ -0,0 +1,653 @@
...
+        public async Task ChangeContactAvatarCommandDoesNothingWhenNoProvider()
+        {
+            var contact = CreateContact("user@site.com", "User");
+            var (vm, _, core, _, _) = CreateVm(new[] { contact });
+            using (vm)
+            {
+                var command = (AsyncRelayCommand<ContactItem>)vm.ChangeContactAvatarCommand;
+                await command.ExecuteAsync(new ContactItem(contact)).ConfigureAwait(false);
+
+                Assert.That(core.SetContactAvatarCalls, Is.EqualTo(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ContactsPanelPageViewModelTests.cs]: You can use `Is.Zero`

File: src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ContactsPanelPageViewModelTests.cs [hunk 6]
```diff
@@ -0,0 +1,653 @@
...
+            var contact = CreateContact("user@site.com", "User");
+            var (vm, _, core, _, _) = CreateVm(new[] { contact });
+            using (vm)
+            {
+                vm.SetAvatarProvider(() => Task.FromResult<byte[]?>(null));
+
+                var command = (AsyncRelayCommand<ContactItem>)vm.ChangeContactAvatarCommand;
+                await command.ExecuteAsync(new ContactItem(contact)).ConfigureAwait(false);
+
+                Assert.That(core.SetContactAvatarCalls, Is.EqualTo(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ContactsPanelPageViewModelTests.cs]: You can use `Is.Zero`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 7]
```diff
@@ -0,0 +1,1287 @@
...
+            // Load initial data
+            var loadResult = await collection.LoadMoreItemsAsync(5).AsTask().ConfigureAwait(false);
+            Assert.That(loadResult.Count, Is.EqualTo(5));
+            Assert.That(collection.Count, Is.EqualTo(5));
+
+            // Refresh after load completes
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+
+            Assert.That(source.ResetCalls, Is.GreaterThanOrEqualTo(1));
+            Assert.That(collection.Count, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Not.Empty);` for any IEnumerable collection, otherwise use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 8]
```diff
@@ -0,0 +1,1287 @@
...
+
+            // Preload some items
+            var preload = await collection.LoadMoreItemsAsync(3).AsTask().ConfigureAwait(false);
+            Assert.That(preload.Count, Is.EqualTo(3));
+            Assert.That(collection.Count, Is.EqualTo(3));
+
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+
+            Assert.That(source.ResetCalls, Is.GreaterThanOrEqualTo(1));
+            Assert.That(collection.Count, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Not.Empty);` for any IEnumerable collection, otherwise use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 9]
```diff
@@ -0,0 +1,1287 @@
...
+            var source = new PagedIntSource(Enumerable.Range(1, 100));
+            var collection = new TestIncrementalLoadingCollection(source, cts);
+
+            // Initial load
+            await collection.LoadMoreItemsAsync(5).AsTask().ConfigureAwait(false);
+            Assert.That(collection.Count, Is.EqualTo(5));
+
+            // Multiple sequential refresh calls
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+            Assert.That(collection.Count, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Not.Empty);` for any IEnumerable collection, otherwise use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 10]
```diff
@@ -0,0 +1,1287 @@
...
+
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+            Assert.That(collection.Count, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Not.Empty);` for any IEnumerable collection, otherwise use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 11]
```diff
@@ -0,0 +1,1287 @@
...
+
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+            Assert.That(collection.Count, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Not.Empty);` for any IEnumerable collection, otherwise use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 12]
```diff
@@ -0,0 +1,1287 @@
...
+            // Initial load
+            await collection.LoadMoreItemsAsync(10).AsTask().ConfigureAwait(false);
+            Assert.That(collection.Count, Is.EqualTo(10));
+            int firstResetCalls = source.ResetCalls;
+
+            // Refresh when not loading
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+
+            Assert.That(source.ResetCalls, Is.GreaterThan(firstResetCalls));
+            Assert.That(collection.Count, Is.GreaterThan(0)); // Should have reloaded
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Not.Empty);` for any IEnumerable collection, otherwise use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 13]
```diff
@@ -0,0 +1,1287 @@
...
+
+            var refreshTask = collection.RefreshForTestAsync();
+
+            source.AllowCompletion();
+
+            await loadTask.ConfigureAwait(false);
+            await refreshTask.ConfigureAwait(false);
+
+            Assert.That(collection.IsLoading, Is.False);
+            Assert.That(collection.Count, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Not.Empty);` for any IEnumerable collection, otherwise use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 14]
```diff
@@ -0,0 +1,1287 @@
...
+            Assert.That(collection.Count, Is.EqualTo(10));
+            Assert.That(collection.OriginalItems.Count, Is.EqualTo(10));
+
+            // Trigger refresh to verify collection was cleared and reloaded
+
+            // Refresh
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+
+            // Collection should be cleared and reloaded (starts from beginning)
+            Assert.That(collection.Count, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Not.Empty);` for any IEnumerable collection, otherwise use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 15]
```diff
@@ -0,0 +1,1287 @@
...
+        [Test]
+        public async Task LoadMoreItemsAsyncCountZeroLoadsDefaultPage()
+        {
+            using var cts = new CancellationTokenSource();
+            var source = new PagedIntSource(Enumerable.Range(1, 100));
+            var collection = new IncrementalLoadingCollection<PagedIntSource, int>(source, cts);
+
+            LoadMoreItemsResult result = await collection.LoadMoreItemsAsync(0).AsTask().ConfigureAwait(false);
+
+            Assert.That(result.Count, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Not.Empty);` for any IEnumerable collection, otherwise use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 16]
```diff
@@ -0,0 +1,1287 @@
...
+        {
+            using var cts = new CancellationTokenSource();
+            var source = new PagedIntSource(Enumerable.Range(1, 50));
+            var collection = new TestIncrementalLoadingCollection(source, cts);
+
+            Assert.That(collection.Count, Is.EqualTo(0));
+
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+
+            Assert.That(collection.Count, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Assert.That(collection, Is.Not.Empty);` for any IEnumerable collection, otherwise use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 17]
```diff
@@ -0,0 +1,1287 @@
...
+                        changingStartCount++;
+                    else
+                        changingEndCount++;
+                }
+            };
+
+            await collection.LoadMoreItemsAsync(5).AsTask().ConfigureAwait(false);
+
+            Assert.That(changingStartCount, Is.GreaterThan(0));
+            Assert.That(changingEndCount, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ManagedCollectionTests/ManagedCollectionTests.ReconcileOriginalItems.cs [hunk 18]
```diff
@@ -0,0 +1,1798 @@
...
+
+            // Act
+            collection.ReconcileOriginalItems(targetList);
+
+            // Assert
+            Assert.That(collection.ToArray(), Is.EqualTo(targetList));
+            Assert.That(collection.OriginalItems, Is.EqualTo(targetList));
+            // Should have removed 3 and added 3
+            Assert.That(removeCount, Is.GreaterThan(0));
+            Assert.That(addCount, Is.GreaterThan(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ManagedCollectionTests/ManagedCollectionTests.ReconcileOriginalItems.cs]: You can use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 19]
```diff
@@ -0,0 +1,1287 @@
...
+            using var cts = new CancellationTokenSource();
+            var source = new PagedIntSource(Enumerable.Range(1, 50));
+            var collection = new TestIncrementalLoadingCollection(source, cts);
+
+            Assert.That(collection.Count, Is.EqualTo(0));
+
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+
+            Assert.That(collection.Count, Is.GreaterThan(0));
+            Assert.That(source.ResetCalls, Is.GreaterThanOrEqualTo(1));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 20]
```diff
@@ -0,0 +1,1287 @@
...
+            var collection = new TestIncrementalLoadingCollection(source, cts);
+
+            // Preload some items
+            var preload = await collection.LoadMoreItemsAsync(3).AsTask().ConfigureAwait(false);
+            Assert.That(preload.Count, Is.EqualTo(3));
+            Assert.That(collection.Count, Is.EqualTo(3));
+
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+
+            Assert.That(source.ResetCalls, Is.GreaterThanOrEqualTo(1));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ManagedCollectionTests/ManagedCollectionTests.ReconcileOriginalItems.cs [hunk 21]
```diff
@@ -0,0 +1,1798 @@
...
+            // Act
+            collection.ReconcileOriginalItems(targetList, (oldItem, newItem) =>
+            {
+                updatesCount++;
+                oldItem.Name = newItem.Name; // Apply update
+            });
+
+            // Assert
+            // A updated, B matched (no update), C added (no update call, just add)
+            Assert.That(updatesCount, Is.GreaterThanOrEqualTo(1));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ManagedCollectionTests/ManagedCollectionTests.ReconcileOriginalItems.cs]: You can use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ManagedCollectionTests/ManagedCollectionTests.StateAndEdgeCases.cs [hunk 22]
```diff
@@ -0,0 +1,673 @@
...
+        {
+            var col = new ManagedCollection<string>();
+            int changeEvents = 0;
+
+            ((INotifyCollectionChanged)col).CollectionChanged += (s, e) => changeEvents++;
+
+            col.Add("A");
+
+            Assert.That(col, Is.EqualTo(new[] { "A" }));
+            Assert.That(changeEvents, Is.GreaterThanOrEqualTo(1));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ManagedCollectionTests/ManagedCollectionTests.StateAndEdgeCases.cs]: You can use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ContactsPanelPageViewModelTests.cs [hunk 23]
```diff
@@ -0,0 +1,653 @@
...
+                });
+
+                core.RaiseUnreadMessagesReceived(new EmailAddress("acc@local"));
+
+                await SpinWaitAsync(
+                    () => collection.OriginalItems.Any(i => i.Email.Address == "a@a.com" && i.UnreadMessagesCount == 5)
+                       && collection.OriginalItems.Any(i => i.Email.Address == "b@b.com" && i.UnreadMessagesCount == 1))
+                    .ConfigureAwait(false);
+
+                Assert.That(core.GetUnreadCountsCalls, Is.GreaterThanOrEqualTo(1));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ContactsPanelPageViewModelTests.cs]: You can use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs [hunk 24]
```diff
@@ -0,0 +1,1287 @@
...
+
+            // Load initial data
+            var loadResult = await collection.LoadMoreItemsAsync(5).AsTask().ConfigureAwait(false);
+            Assert.That(loadResult.Count, Is.EqualTo(5));
+            Assert.That(collection.Count, Is.EqualTo(5));
+
+            // Refresh after load completes
+            await collection.RefreshForTestAsync().ConfigureAwait(false);
+
+            Assert.That(source.ResetCalls, Is.GreaterThanOrEqualTo(1));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.UI.Tests/IncrementalLoadingCollectionTests.cs]: You can use `Is.Positive`

File: src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ContactsPanelPageViewModelTests.cs [hunk 25]
```diff
@@ -0,0 +1,653 @@
...
+            var (vm, _, _, settings, _) = CreateVm();
+            using (vm)
+            {
+                settings.ContactsSortOrder = ContactsSortOrder.ByName;
+
+                vm.OnNavigatedTo(null);
+
+                Assert.That(vm.SortingVariants, Is.Not.Null);
+                Assert.That(vm.SortingVariants.Length, Is.EqualTo(3));
+                Assert.That(vm.SelectedSortingIndex, Is.GreaterThanOrEqualTo(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ContactsPanelPageViewModelTests.cs]: You can use `Is.Not.Negative` or `Is.Zero.OrPositive`

File: src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ContactsPanelPageViewModelTests.cs [hunk 26]
```diff
@@ -0,0 +1,653 @@
...
+        {
+            var (vm, collection, _, settings, _) = CreateVm();
+            using (vm)
+            {
+                vm.OnNavigatedTo(null);
+
+                var initialRefresh = collection.RefreshCalls;
+
+                var byNameIdx = Array.FindIndex(vm.SortingVariants, v => v.SortOrder == ContactsSortOrder.ByName);
+                Assert.That(byNameIdx, Is.GreaterThanOrEqualTo(0));
```
Comment (al-kau) [inline src/Eppie.App/Tests/Eppie.App.ViewModels.Tests/ContactsPanelPageViewModelTests.cs]: You can use `Is.Not.Negative` or `Is.Zero.OrPositive`
